lib/Util/RequestOptions.php

```diff
@@ -25,6 +25,21 @@ public function __construct($key = null, $headers = [], $base = null)
         $this->apiBase = $base;
     }
 
+    public function __debugInfo()
+    {
+        $redactedApiKey = strlen($this->apiKey) >= 12
+            ? (substr($this->apiKey, 0, 8)
+                . str_repeat('*', strlen($this->apiKey) - 12)
+                . substr($this->apiKey, -4))
+            : $this->apiKey;
```

Comment:
While that works, legacy keys didn't start with `sk_test` or `sk_live`. Can we just redact all but last 4 if they don't start with one of those?

Reply:
Fixed.